deploy/olm-catalog/kogito-cloud-operator/0.6.0/kogito-cloud-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -46,7 +46,7 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     certified: "false"
-    containerImage: quay.io/kiegroup/kogito-cloud-operator:0.6.0
+    containerImage: quay.io/kiegroup/kogito-cloud-operator:0.5.0
```

Comment:
isn't this wrong? at least is mismatched with the file/directory name

Reply:
That's the auto-generated CSV that will be fixed later in another PR.